src/libraries/System.Net.Quic/tests/FunctionalTests/QuicTestCollection.cs

```diff
@@ -33,11 +34,28 @@ public QuicTestCollection()
             QUIC_SETTINGS settings = default(QUIC_SETTINGS);
             settings.IsSet.MaxWorkerQueueDelayUs = 1;
             settings.MaxWorkerQueueDelayUs = 2_500_000u; // 2.5s, 10x the default
-            if (MsQuic.StatusFailed(GetApiTable()->SetParam(null, MsQuic.QUIC_PARAM_GLOBAL_SETTINGS, (uint)sizeof(QUIC_SETTINGS), (byte*)&settings)))
+            if (StatusFailed(GetApiTable()->SetParam(null, QUIC_PARAM_GLOBAL_SETTINGS, (uint)sizeof(QUIC_SETTINGS), (byte*)&settings)))
             {
                 Console.WriteLine($"Unable to set MsQuic MaxWorkerQueueDelayUs.");
             }
         }
+
+        EventHandler<UnobservedTaskExceptionEventArgs> eventHandler = (_, e) =>
+        {
+            lock (s_unobservedExceptions)
+            {
+                string text = e.Exception.ToString();
+                if (!s_unobservedExceptions.ContainsKey(text))
+                {
+                    s_unobservedExceptions[text] = 1;
+                }
+                else
+                {
+                    s_unobservedExceptions[text] += 1;
+                }
```

Comment:
Perf doesn't matter here, but for reference you can do the same with
```suggestion
                string text = e.Exception.ToString();
                CollectionsMarshal.GetValueRefOrAddDefault(s_unobservedExceptions, text, out _)++;
```

Reply:
or `s_unobservedExceptions[text] = s_unobservedExceptions.GetValueOrDefault(text) + 1;`

Reply:
Fixed